ros/api/v0/hosts.py

```diff
@@ -17,6 +17,11 @@
 SYSTEM_STATES_EXCEPT_OPTIMIZED = [
     "Oversized", "Undersized", "Idling", "Storage rightsizing"
 ]
+system_columns = [
```

Comment:
```suggestion
SYSTEM_COLUMNS = [
```
Constants are written in all capital letters & underscores separating the words.